code/analytics/src/androidTest/java/com/adobe/marketing/mobile/analytics/internal/MonitorExtension.kt

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright 2022 Adobe. All rights reserved.
```

Comment:
It's best if we keep the original copyright date.

Reply:
In this case the file already had a copyright notice, so we don't need to change the date. If we copy a file from another repo, we can keep that copyright date. However, if the copied file does not have a copyright notice then we can use the date it was added to the new repo. That's my two cents at least.